kibana/config.sls

```diff
@@ -21,3 +25,5 @@ kibana-config:
     - group: root
     - dataset: {{ kibana.config }}
     - formatter: "YAML"
+    - require_in:
```

Comment:
This should be `require`, and then you won't need your additional state above this one: `kibana-create-config-folder`.

Reply:
@blbradley oh yes sorry, you are right. I just did a test install in a docker container with the source package. I thought that if you install it from source it may not create the required folder. But it does create it. I guess #13 is some weird issue.

Reply:
No problem! I appreciate your patience and acceptance of my help. I saw the storm of commits this morning in my inbox (not an issue at all). Felt like I could use this as a teaching experience.

Reply:
Haha, yeah I am really sorry for the noise that the commits created. I thought I had to create a commit for each change I did, so everyone could see my trial and error experience, especially regarding the use of jinja2 "startswith" stuff. I now get that I'll only create a commit if it's building and working in my local kitchen setup.

I am still struggeling with the information I should put into commits.

Thank you very much for time and also @johnkeates as I did a whole lot of mess here.

Reply:
@philippkahr Don't worry about it too much; it's more of an art than a strict set of rules.